assets/js/main.js

```diff
@@ -178,3 +87,108 @@ $(document).ready(
                 });
             });
     });
+
+// Export comment functionality and include functionality for comment count button
+function commentFunction() {
+            $('#js-expander-trigger').toggleClass("expander-hidden");
+
+
+    // Create variable for request URI.
+            var commentServer = "{{ site.comment_server_url }}";
```

Comment:
@flowersw Same thing, this line and the lines below are indented too many spaces